ipinfo/main.go

```diff
@@ -73,6 +73,8 @@ func main() {
 		err = cmdQuota()
 	case cmd == "login":
 		err = cmdLogin()
+	case cmd == "init":
```

Comment:
We need to remove `login` now. I'd leave `logout` as it's useful for debugging stuff without a token if you had one previously.